docs/guides/docs/running-a-node/running-a-beta-4-node.mdx

```diff
@@ -0,0 +1,304 @@
+---
+title: Running a Beta-4 Node
+category: Running a Node
+parent:
+  label: Guides
+  link: /guides
+---
+
+# Running a local Fuel node connected to Beta-4 using P2P
+
+## Installation
+Make sure you have the latest version of fuelup by running the following command:
```

Comment:
It might be useful to new comers to mention how to get `fuelup`, or to link to installation instructions.

Reply:
Good call! Added for now but will have to consolidate all installations later to avoid duplicates.